pkg/kubelet/apis/config/v1beta1/defaults.go

```diff
@@ -71,6 +71,9 @@ func SetDefaults_KubeletConfiguration(obj *kubeletconfigv1beta1.KubeletConfigura
 	if obj.Port == 0 {
 		obj.Port = ports.KubeletPort
 	}
+	if obj.ReadOnlyBindAddress == "" {
+		obj.ReadOnlyBindAddress = obj.Address
```

Comment:
Do any of the other defaulters in this file refer to `obj` on the right-hand-side of any defaulting expression? If not, then introducing `obj` in the RHS here may add an implicit sequencing requirement that didn't exist before (e.g. if there's a default `obj.Address`, then that _must_ be applied before `obj.ReadOnlyBindAddress` is defaulted). Might be something we want to be careful with, but for this it's probably also fine to add a comment since `Address` and `ReadOnlyBindAddress` are so close together here.

Reply:
hmm. defaulting here seemed icky to me as well. I can move this to just before we call `ListenAndServeReadOnly`.

Reply:
moved it to the ListenAndServeReadOnly call site.